app/src/test/java/de/westnordost/streetcomplete/quests/address/AddAddressStreetTest.kt

```diff
@@ -18,7 +18,7 @@ class AddAddressStreetTest {
         val addr = node(tags = mapOf("addr:housenumber" to "123"))
         val mapData = TestMapDataWithGeometry(listOf(addr))
         assertEquals(1, questType.getApplicableElements(mapData).toList().size)
-        assertNull(questType.isApplicableTo(addr))
+        assertNull(questType.isApplicableTo(addr,))
```

Comment:
What happened here? :open_mouth: 
I think most formatting changes in the test files could be completely reversed.

Reply:
> Also, what's missing is to adapt the tests, so a final PR would add a few hundred lines of code more.